manifests/0000_90_cluster-etcd-operator_02-dashboards.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
also another nitpick, why not put it to the existing dashboard in jsonnet?
https://github.com/openshift/cluster-etcd-operator/blob/master/jsonnet/dashboard.jsonnet

Reply:
+1, this is the best option .. i will change it now 👍🏽

Reply:
Done 👍🏽

Reply:
Thanks. Strange that the verify job doesn't complain, doesn't this need to go into the binary with `make update`?

Reply:
i ran `make update`, `make verify` !! 

let me reassure 👍🏽

Reply:
ran them again, nothing changed :/